doc/changelog/04-tactics/13741-remove_omega.rst

```diff
@@ -0,0 +1,5 @@
+- **Removed:**
+  Removed the `omega` tactic (deprecated in 8.12) and 4 `* Omega *` flags.
+  Use `lia` instead.
+  (`#13741 <https://github.com/coq/coq/pull/13741>`_,
+  by Jim Fehrle).
```

Comment:
Maybe it would be worth expanding this changelog entry to remind that this is the end of a long-term effort with many people involved (especially Frédéric Besson who improved `lia` a lot for this to happen, Maxime Dénès who initiated the effort, Vincent Laporte who contributed a lot and many other people that cannot be listed exhaustively, including many maintainers of Coq projects who tried to migrate and reported the issues that they found).

Reply:
How about this?

  by Jim Fehrle with the help of
  Frédéric Besson, who greatly improved `lia`, Maxime Dénès,
  Vincent Laporte and many package maintainers, among others).

Reply:
Or this?

  by Jim Fehrle, who addressed the final details, building on much work by
  Frédéric Besson, who greatly improved `lia`, Maxime Dénès,
  Vincent Laporte and with the help of many package maintainers, among others).

Reply:
Yes, the second one looks very good.